prepare

```diff
@@ -105,6 +105,11 @@ remove_stale() {
 }
 
 main() {
+  # When using local blobs, its useful to start with an empty slate
+  if [ $CLEAN_BLOBS ]; then
+    > config/blobs.yml
```

Comment:
This is the safest way to empty a file?

Reply:
It was a quick way 😉Whats nice about this is that it leaves the file existing and keeps its permissions. 
Open to alternatives though